app/views/projects/dataset/keys/_boolean_key.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="dataset-key boolean-checkbox">
-  <%= f.check_box :value %>
+<div class="dataset-key boolean-select">
   <%= f.label :value, key.title %>
+  <%= f.select :value, [[_('No'), 0], [_('Yes'), 1]], include_blank: true %>
```

Comment:
Much nicer! 👍 

Should add a `required` attribute to this to make sure someone selects it?

Reply:
Decided against this. In our initial use case we're asking:

1. Is there anything of interest in this request?
2. Summarise the information released
3. Notes

You can still summarise the information even if you're not sure whether it's interesting or not.